alternator/conditions.cc

```diff
@@ -239,20 +248,23 @@ static bool verify_expected_one(const rjson::value& condition, const rjson::valu
         comparison_operator_type op = get_comparison_operator(*comparison_operator);
         switch (op) {
         case comparison_operator_type::EQ:
-            verify_operand_count(attribute_value_list, 1, *comparison_operator);
+            verify_operand_count(attribute_value_list, exact_size(1), *comparison_operator);
```

Comment:
I like this. A nice switch where you can see all the logic - and which functions get called - in one place.